virtual-setup/windows-instructions.md

```diff
@@ -1,7 +1,75 @@
 
-### Zoom
+- [Windows set up instructions for virtual workshops](#windows-set-up-instructions-for-virtual-workshops)
+  - [Zoom Installation and Setup](#zoom-installation-and-setup)
+    - [New Installation](#new-installation)
+    - [Set Up Preferences](#set-up-preferences)
+  - [Slack](#slack)
 
-### Slack
\ No newline at end of file
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Zoom Installation and Setup
+
+### New Installation
+
+If you do not have Zoom installed yet, you will need to download the client.
+To do so, go to https://zoom.us/download and click the button to download the latest version of **Zoom Client for Meetings**
+
+<img src="screenshots/windows-zoom-download.png" width=500 alt="Zoom Download Page">
+
+The `Zoominstaller.exe` installer package will most likely be saved to your `Downloads` folder or you can click on it in the lower left corner.
+Click `Run` when asked what you would like to do with `Zoominstaller.exe`
+
+Windows will ask you if you would like to allow Zoom to make changes to your device. Click `Yes`.
+
+When the installation is completed, Zoom should automatically open.
+Click `Join with Video` if asked.
+
+<img src="screenshots/windows-zoomsplash.png" alt="Zoom splash screen" width=400>
+
+Login or create a new Zoom account.
+
+<img src="screenshots/windows-zoom-sign-in.png" alt="Zoom sign in screen" width=400>
+
+### Set Up Preferences
+
+Click on the gear in the corner of the main Zoom page menu.
+
+<img src="screenshots/windows-zoom-preferences.png" alt="Zoom preferences menu" width=1000>
+
+If it is not selected already, click on the **Audio** panel in the left sidebar.
+
+<img src="screenshots/windows-audio.png" alt="Microphone preferences" width=500>
+
+If you have granted permission for Zoom to use the microphone, you should now see green and/or red bars in the *Input Level* section as you talk.
+If you want to further test your microphone and speaker settings, you can click the *Test Speaker* and *Test Mic* buttons.
+
+We recommend that you select the checkbox to "Automatically join audio by computer when joining a meeting." to save you the future annoyance of having to do this every time you join a meeting.
+
+## Slack
+
+Download 64-bit Slack application from:
+https://slack.com/downloads/instructions/windows
+
+<img src="screenshots/windows-slack-download.png" alt="Slack download" width=1000>
+
+Click on the `SlackSetUp.exe` file this downloads when it pops up in the lower left corner.
+Or find it most likely in the `Downloads` folder and click on it there.
+
+Sign in with your Slack account if you have one, or create a new account.
+
+<img src="screenshots/windows-slack-sign-in.png" width=500 alt="Slack sign in page">
+
+Join Cancer Data Science Slack by using [this link](https://ccdatalab.org/slack)
+
+Or if you already have a Slack account, when prompted, you can type in `cancer-data-science.slack.com` as your workspace URL.
```

Comment:
To be clear, you need to have a **Cancer Data Science Community*** account; if you have one for another Slack workspace, that won't work here. (This is a strange and confusing Slack decision: there is no such thing as a universal "Slack Account", though they do tie them together by shared email addresses... sometimes.)

Reply:
Yes. This is confusing. What should I write here instead them? I too am confused.

Reply:
I just took your wording where applicable.

Reply:
I think the language about logging in within the app should be removed. It only works sometimes, whereas going to ccdatalab/slack always works.